docs/quick_start.rst

```diff
@@ -0,0 +1,132 @@
+Quick Start
+===========
+
+Detect Action Potentials
+------------------------
+
+To detect action potentials, first construct three 1D numpy arrays: stimulus current ``i`` in pA,
+response voltage ``v`` in mV, and timestamps ``t`` in seconds.  With these three arrays, you can then do the following:
+
+.. code-block:: python
+
+    from ipfx.feature_extractor import SpikeFeatureExtractor
+
+    ext = SpikeFeatureExtractor()
+    spikes = ext.process(t, v, i)
+
+The ``spikes`` object is a ``pandas`` ``DataFrame`` where rows are action potentials and columns are features.
+
+
+Extract Spike Train Features
+----------------------------
+
+Given a spike train ``DataFrame``, you can then compute a number of other features(e.g, adaptation and latency:
+
+.. code-block:: python
+
+    from ipfx.feature_extractor import SpikeTrainFeatureExtractor
+
+    ext = SpikeTrainFeatureExtractor()
+    features = ext.process(t, v, i, spikes) # re-using spikes from above
+
+
+Stimulus-specific Analysis
+--------------------------
+
+To analyze all of the sweeps with a particular stimulus type (say, long square pulses), you'll first need to create
+a :py:class:`~ipfx.sweep.SweepSet` object. This object provides utilities for accessing properties of a group
+of :py:class:`~ipfx.sweep.Sweep` objects:
+
+.. code-block:: python
+
+    from ipfx.sweep import Sweep, SweepSet
+
+    sweep_set = SweepSet([ Sweep(t=t0, v=v0, i=i0),
+                           Sweep(t=t1, v=v1, i=i1),
+                           Sweep(t=t2, v=v2, i=i2) ])
+
+Now that we have this object, we can hand it to one of the stimulus-specific analysis classes.  You first need
+to configure a :py:class:`~ipfx.feature_extractor.SpikeFeatureExtractor` and :py:class:`~ipfx.feature_extractor.SpikeTrainFeatureExtractor`:
+
+.. code-block:: python
+
+    from ipfx.feature_extractor import SpikeFeatureExtractor, SpikeTrainFeatureExtractor
+    from ipfx.stimulus_protocol_analysis import LongSquareAnalysis
+
+    start, end = 1.02, 2.02 # start/end of stimulus in seconds
```

Comment:
how would one obtain this information?

Reply:
good point, added getting this from the epochs.